### ISSUE TITLE
allow setting integers to unsigned

### ISSUE THREAD

**Issue by eshell** [balderdashy/sails-mysql#98]
any chance you can allow setting integers to unsigned? I know it's not a priority but would be a nice feature. I've been trying to create a PR to do this, but my versions just end up setting the field to varchar(255) or the `id` to varchar(255). 

Tried changing sql.js file such as..
```
case "int":
  if(attr.unsigned)
    return 'INT UNSIGNED';
  else
    return 'INT';
```
This just causes the field:{type:'int',unsigned:true} to become `varchar(255)`.

**Comment by sailsbot** [comment]
Thanks for posting, @eshell.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
